dashboards/TDinsight.md

```diff
@@ -349,6 +349,22 @@ Here's the metrics list:
 
 Currently only report login count per minute.
 
+## Upgrade
+
+Re install with `TDinsight.sh` will upgrade Grafana plugin and TDinsight dashboard.
```

Comment:
Reinstall

Reply:
TDinsight dashboard(s)

Reply:
I prefer to use dashboard as it specificly only one.